jax/_src/scipy/special.py

```diff
@@ -657,3 +659,92 @@ def i1e(x):
 def i1(x):
   x, = _promote_args_inexact("i1", x)
   return lax.mul(lax.exp(lax.abs(x)), lax.bessel_i1e(x))
+
+def lpmn(m, n, z):
+  r"""The normalized associated Legendre functions of the first kind.
+
+  In the descriptions of the associated Legendre functions (ALFs), we use `l`
+  to denote the degree (`l_max` is the maximum degree) and `m` to denote
+  the order. The ALFs of the first kind are often used in spherical harmonics.
+  The spherical harmonic of degree `l` and order `m` can be written as
+  .. math::
+    \begin{equation}
```

Comment:
I don't think the `\begin{equation}` blocks are necessary here & below

Reply:
Removed the \begin{equation} blocks here and after.